[38;21m2025-09-29 22:06:31,298 :: DEBUG :: icloudpy.base :: base.py :: 240 :: Using session file /tmp/icloudpy/runner/quentintarantinohotmailfr.session[0m
[38;5;39m2025-09-29 22:06:31,299 :: INFO :: icloudpy.base :: base.py :: 247 :: Session file does not exist[0m
[38;21m2025-09-29 22:06:31,299 :: DEBUG :: icloudpy.base :: base.py :: 313 :: Authenticating as quentintarantino@hotmail.fr[0m
[38;21m2025-09-29 22:06:31,308 :: DEBUG :: icloudpy.base :: base.py :: 417 :: Authentication completed successfully[0m
[38;21m2025-09-29 22:06:31,326 :: DEBUG :: root :: config_parser.py :: 246 :: max_threads is not found in app > max_threads. Using default max_threads: 4 (auto) ...[0m
